### PR TITLE
Bump up version to 0.3.1

### PR DESCRIPTION
**Pull request by rantan**

### CODE REVIEW

File: doc/release-notes/release-notes-0.3.1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Tapyrus Signer version 0.3.1 is now available for download at: 
+  https://github.com/chaintope/tapyrus-signer/release/tag/v0.3.1/
```
Comment (Yamaguchi) [inline doc/release-notes/release-notes-0.3.1.md]: I could not download from this url.

File: doc/release-notes/release-notes-0.3.1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    
+Please report bugs using the issue tracker at github:
+  https://github.com/chaintope/tapyrus-signer/issues
+  
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+  https://github.com/chaintope/tapyrus-signer/tarball/v0.3.1  # .tar.gz
```
Comment (Yamaguchi) [inline doc/release-notes/release-notes-0.3.1.md]: I could not download from this url.

Comment (rantan) [reply doc/release-notes/release-notes-0.3.1.md]: Yes, i've not created yet. I'm going to create binary archive and tag after RP #52, #53 will be merged.

**Comment by rantan** [comment]
I added 1 commit for to update release process document because of it was old.

**Pull request merged**